src/main/java/mate/academy/Main.java

```diff
@@ -1,16 +1,42 @@
 package mate.academy;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import mate.academy.lib.Injector;
+import mate.academy.model.CinemaHall;
 import mate.academy.model.Movie;
+import mate.academy.model.MovieSession;
+import mate.academy.service.CinemaHallService;
 import mate.academy.service.MovieService;
+import mate.academy.service.MovieSessionService;
 
 public class Main {
+
     public static void main(String[] args) {
-        MovieService movieService = null;
+        Injector injector = Injector.getInstance("mate.academy");
+        CinemaHallService cinemaHallService = (CinemaHallService)
+                injector.getInstance(CinemaHallService.class);
+
+        CinemaHall cinemaHall = new CinemaHall();
+        cinemaHall.setCapacity(15);
+        cinemaHall.setDescription("test");
+        cinemaHallService.add(cinemaHall);
 
+        MovieService movieService = (MovieService) injector.getInstance(MovieService.class);
         Movie fastAndFurious = new Movie("Fast and Furious");
         fastAndFurious.setDescription("An action film about street racing, heists, and spies.");
         movieService.add(fastAndFurious);
-        System.out.println(movieService.get(fastAndFurious.getId()));
-        movieService.getAll().forEach(System.out::println);
+
+        MovieSession movieSession = new MovieSession();
+        movieSession.setMovie(fastAndFurious);
+        movieSession.setCinemaHall(cinemaHall);
+        String date = "2024-12-19";
+        DateTimeFormatter dateFormatter = DateTimeFormatter.ofPattern("yyyy-MM-dd");
+        LocalDate localDate = LocalDate.parse(date, dateFormatter);
+        movieSession.setShowTime(LocalDate.parse("2024-11-19", dateFormatter));
```

Comment:
You have parsed the date into `localDate` but then directly parse the date string again when setting the show time for `movieSession`. Consider using `localDate` instead for consistency and to avoid redundant parsing.